pyincore/dfr3service.py

```diff
@@ -175,23 +177,32 @@ def create_dfr3_set(self, dfr3_set: dict, timeout=(30, 600), **kwargs):
         r = self.client.post(url, json=dfr3_set, timeout=timeout, **kwargs)
         return return_http_response(r).json()
 
-    def match_inventory(self, mapping: MappingSet, inventories: list, entry_key: str, add_info: list = None):
+    def match_inventory(self, mapping: MappingSet, inventories: list, entry_key: str = None, add_info: list = None):
```

Comment:
Just a suggestion: you can add Optional[str] from typing here. Doesn't make a huge difference but it shows up in code suggestions when someone uses it.